Scripts/symbolic_hermitians/HermitianUtils.py

```diff
@@ -93,6 +93,12 @@ def SU_vector_magnitude(self):
 
         return sympy.sqrt(mag2)
     
```

Comment:
The following is unused. It was a byproduct of my other tinkering, but I didn't see a reason to delete it.